monkey/monkey_island/cc/ui/src/components/ui-components/HideInput.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {InputGroup, FormControl} from 'react-bootstrap';
+import '../../styles/components/HideInput.scss'
+
+class HideInput extends React.PureComponent {
```

Comment:
Maybe a name like `SensitiveTextInput` would be better? The input isn't always hidden.